docs/debug.md

```diff
@@ -39,7 +39,7 @@ $ kubectl port-forward --namespace gitops-postgresql svc/gitops-postgresql-stagi
 Then run a `psql` command, to verify your access:
 
 ```shell
-psql postgresql://postgres:$PASSWORD@127.0.0.1:5432/postgres -c "select now()"
+psql postgresql://postgres:$PASSWORD@127.0.0.1:5432/$POSTGRESQL_DATABASE -c "select now()"
```

Comment:
Before this line, it couldn't hurt to say that POSTGRESQL_DATABASE defaults to `postgres`, so that users new to the project know what value to put in this field.